pages/api/Schema/ContactFilters/datahandler.ts

```diff
@@ -0,0 +1,67 @@
+import {
```

Comment:
I was debating if this should live in its own separate file called ```dataHandler.ts``` (let me know if you can think of a better name), or if it should live in ```resolvers.ts```. I'm leaning more towards a separate file to help separate concerns but let me know what you guys think.

Reply:
Yeah I think separate is good... it looks like graphql-rest.page.ts uses this file, it's probably easier to import this into more places by keeping it separate from resolvers.ts.